homepage/homepage/components/home/ChatDemoSection.tsx

```diff
@@ -133,7 +133,7 @@ export function ChatDemoSection() {
         title="See it for yourself"
         slogan={
           <>
-            A chat app in 174 lines of client-side code.{" "}
+            A chat app with image upload in 300+ lines of client-side code.{" "}
```

Comment:
Maybe would be better to say `~300 lines`?

300+ means that the lines could be way more

Reply:
yeah you're right